supervisor/homeassistant/const.py

```diff
@@ -7,7 +7,11 @@
 
 LANDINGPAGE: AwesomeVersion = AwesomeVersion("landingpage")
 
-MIN_VERSION = {"supervisor/event": "2021.2.4"}
+MIN_VERSION = {
+    "supervisor/event": "2021.2.4",
+    "backup/start": "2021.12.0",
+    "backup/end": "2021.12.0",
+}
```

Comment:
@ludeeus I think we use WSType here too, thoughts?

Reply:
We should

Reply:
Needs a bit an awkward reordering (constants, class, constants) but :man_shrugging: 
https://github.com/home-assistant/supervisor/pull/3305/commits/5270bb3ce506001b49d719d1125f3d7fd7c4ccb1

Reply:
`MIN_VERSION` is only used in `supervisor/homeassistant/websocket.py` we can move that there